ipython-extension/setup.py

```diff
@@ -19,7 +19,7 @@ def get_long_description():
     keywords=["jupyter", "jupyterlab", "matplotlib", "mpld3", "time series"],
     packages=find_packages(include=["autoplot", "autoplot.*"], exclude=["tests", "tests.*"]),
     include_package_data=True,
-    install_requires=["ipywidgets", "ipython", "numpy", "pandas", "matplotlib", "mpld3", "dtale>=1.16.0,<1.36"],
+    install_requires=["ipywidgets", "ipython", "numpy", "pandas", "matplotlib", "mpld3", "dtale>=1.16.0,<1.36,>=2.15.2"],
```

Comment:
Should this just be `"dtale>=2.15.2"` now?

I *think* this is the spec for that version requirement syntax https://peps.python.org/pep-0440/#version-specifiers. And they say:

> The comma (“,”) is equivalent to a logical and operator: a candidate version must match all given version clauses in order to match the specifier as a whole.

So it sounds like this set of requirements would be impossible (but correct me if I'm wrong, I haven't actually tried it out)

Reply:
Ahh, good catch. Sorry I thought the comma was treated like an or-clause and not an and-clause.  I think I'll update it to `>=2.15.2` just because I don't know if there was something bad with versions `>1.36,<2.15.1`